app/controllers/idv/scan_id_base_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  class ScanIdBaseController < ApplicationController
```

Comment:
I could.  The reason I'm not a fan is I don't envision it being used by any other controllers because it's so specific to these two controllers.  However. I don't feel strongly about it.